apps/aecore/lib/aecore/chain/block_validation.ex

```diff
@@ -112,7 +113,12 @@ defmodule Aecore.Chain.BlockValidation do
   @spec validate_transaction_chainstate(SignedTx.t(), ChainState.account_chainstate(), integer()) :: {boolean(), map()}
   defp validate_transaction_chainstate(tx, chain_state, block_height) do
     try do
-      {true, ChainState.apply_transaction_on_state!(tx, chain_state, block_height)}
+      case tx do
```

Comment:
why is here a differentiation? you could just not use the `block_height` inside the function

Reply:
I would have to make a really big case, I think it's better this way